stack-nodejs-16/build.Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
+ENV CNB_USER_ID=1001
+ENV CNB_GROUP_ID=1000
+ENV CNB_STACK_ID="io.buildpacks.stacks.ubi8"
+ENV CNB_STACK_DESC="ubi nodejs minimal run image base"
```

Comment:
I wonder if there's a way around copying the build.Dockerfile X number of times for all the variants since they're identical.

Reply:
Good point, I pushed a commit to re-use the copy in  the stack directory for the -16 and -18 stack toml files and removed the duplicate copies.